lms/static/scripts/frontend_apps/index.js

```diff
@@ -13,6 +13,10 @@ import { startRpcServer } from '../postmessage_json_rpc/server';
 const rootEl = document.querySelector('#app');
 const config = JSON.parse(document.querySelector('.js-config').textContent);
 
+import { registerIcons } from './components/SvgIcon';
+import iconSet from './icons';
+registerIcons(iconSet);
```

Comment:
This follows the pattern in `src/sidebar/index.js` and `src/annotator/index.js` in the client repo.